frontend/src/templates/Field/Table/TableField.tsx

```diff
@@ -139,6 +142,13 @@ export const TableField = ({
           </Tbody>
         </Table>
       </Box>
+      {isMobile ? null : (
```

Comment:
do we not show the error in mobile?

Reply:
saw the code in `ColumnCell` - would be good to add a comment here to explain!

Reply:
2da28ab